.changeset/early-experts-bathe.md

```diff
@@ -0,0 +1,8 @@
+---
+'@vanilla-extract/esbuild-plugin': minor
+'@vanilla-extract/vite-plugin': minor
+'@vanilla-extract/rollup-plugin': minor
+'@vanilla-extract/next-plugin': minor
```

Comment:
I think maybe Next should get its own changeset since the release notes should be more targeted towards migrating away from the Babel plugin, whereas the other integrations it's just a new feature.

Reply:
Merged the next one into the webpack one.